### DISCUSSION TITLE
How to deploy JanusGraph documents offline

### DISCUSSION THREAD

**Discussion by SwimSweet** [JanusGraph/janusgraph#4277] [Q&A]
How to deploy JanusGraph documents offline

**Comment by hadoopmarc** [comment]
If you download the documents as zip from [https://github.com/JanusGraph/docs.janusgraph.org](https://github.com/JanusGraph/docs.janusgraph.org) it almost works by opening the index.html in the browser. You have additional clicks to local index.html files on each subpage. If you want to avoid these additional clicks you have to serve the pages with some local web server (e.g. flask).
